src/components/designer/taro/actions/MintAssetButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { WalletOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button, Form } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { TaroNode } from 'shared/types';
+import { useStoreActions } from 'store';
+
+const Styled = {
+  Button: styled(Button)`
+    width: 50%;
+  `,
+};
+interface Props {
+  node: TaroNode;
+  type?: 'button' | 'menu';
+}
+
+const MintAssetButton: React.FC<Props> = ({ node, type }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.taro.ActionsTab.MintAssetButton');
```

Comment:
The key should be `cmps.designer.taro.actions.MintAssetButton` to represent the file path to the component.

Reply:
Done